modules/dpu-enable-dpu-hw-offloading.adoc

```diff
@@ -0,0 +1,330 @@
+----
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc create -f ovnkubeconfigmap.yaml
+----
+
+. Label the DPUs in the infrastructure cluster where you want to enable hardware offloading.
++
+[source,terminal]
+----
+$ oc label node <NODENAME> network.operator.openshift.io/dpu=
+----
+
+== Configuring support for hardware offloading in the tenant cluster
+
+Configure support for hardware offloading in the tenant cluster by using this procedure.
+
+
+.Procedure
+
+. Create a MachineConfigPool for all the DPU workers. Save the YAML in the `dputenantmachineconfig.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+name: dpu-host
+spec:
+  machineConfigSelector:
+    matchExpressions:
+    - key: machineconfiguration.openshift.io/role
+      operator: In
+      values:
+      - worker
+      - dpu-host
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/dpu-host: ""
+----
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc create -f dputenantmachineconfig.yaml
+----
+
+. Label the DPU host nodes:
++
+[source,terminal]
+----
+$ oc label node <NODENAME> node-role.kubernetes.io/dpu-host=
+----
+
+. Install and configure the SR-IOV Network Operator:
+
+.. In the {product-title} web console, click *Administration* -> *Namespaces*.
+
+.. Click *Create Namespace*.
+
+.. In the *Name* field, enter `openshift-sriov-network-operator`, and click *Create*.
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Select *SR-IOV Network Operator* from the list of available Operators, and click *Install*.
+
+.. On the *Install Operator* page, under *A specific namespace on the cluster*, select *openshift-sriov-network-operator*.
+
+.. Click *Install*.
+
+.. Verify that the SR-IOV Network Operator is installed successfully:
+
+... Navigate to the *Operators* -> *Installed Operators* page.
+
+... Ensure that *SR-IOV Network Operator* is listed in the *openshift-sriov-network-operator* project with a *Status* of *InstallSucceeded*.
++
+[NOTE]
+====
+During installation an Operator might display a *Failed* status.
+If the installation later succeeds with an *InstallSucceeded* message, you can ignore the *Failed* message.
+====
++
+If the operator does not appear as installed, to troubleshoot further:
+
++
+* Inspect the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors under *Status*.
+* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the
+`openshift-sriov-network-operator` project.
+
+. Add this machine config pool to the SriovNetworkPoolConfig custom resource. 
+
+.. Create a file, such as `sriov-pool-config.yaml`, with the following content:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkPoolConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+spec:
+  ovsHardwareOffloadConfig:
+    name: dpu-host
```

Comment:
It would be better to also mention that the `name` shall be the same as the MCP name we just create.